modules/installing-rhv-creating-custom-vm.adoc

```diff
@@ -0,0 +1,179 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-default.adoc
+// * installing/installing_rhv/installing-rhv-customizations.adoc
+
+[id="installing-rhv-creating-custom-vm_{context}"]
+= Creating a custom virtual machine template for {rh-virtualization}
+
+[NOTE]
+====
+This task is a workaround for a known issue, link:https://bugzilla.redhat.com/show_bug.cgi?id=1818577[bz#1818577]. Red Hat developers are working to resolve this issue. When the issue is resolved, this task will be removed or become optional.
+====
+
+Usually, the installation program downloads an image and uses it to create the cluster nodes. However, in this workaround, you create a custom virtual machine template and configure your environment so the installation program uses this template.
+
+This process includes the following tasks:
+
+. Download a specific Red Hat CoreOS (RHCOS) image to the {rh-virtualization-first} Manager machine.
+. Use an Ansible playbook to upload an RHCOS image to a data storage domain.
+. Attach the virtual disk with the RHCOS image to the virtual machine.
+. Continue configuring the virtual machine and create it.
+. Create a virtual machine template.
+. Export some environment variables.
+
+*Procedure*
+
+.Download a specific Red Hat CoreOS (RHCOS) image to the {rh-virtualization-first} Manager machine
```

Comment:
There's so much information in this module. I don't think there's a good way to present it. I think it would be more correct to convert the subheadings into substeps or create a separate temporary assembly for this set of procedures.

Reply:
> more correct to convert the subheading

How does this look?